mllib/src/main/scala/org/apache/spark/ml/tuning/CrossValidator.scala

```diff
@@ -140,7 +140,11 @@ class CrossValidator(override val uid: String) extends Estimator[CrossValidatorM
 @Experimental
 class CrossValidatorModel private[ml] (
     override val uid: String,
+
+    /** The best model selected from cross validation */
```

Comment:
@jkbradley I don't fully understand. As far as I understand, @param is for documenting arguments of methods. However, both bestModel and avgMetrics (that Spark-9798 are about) are not arguments in any methods (including constructors). Why should there be @params for the two members for the class?  Could you please help elaborate?